### DISCUSSION TITLE
What does 'Trakt service sync interval (in minutes)' do?

### DISCUSSION THREAD

**Discussion by Ade-AV** [umbrellaplug/umbrellaplug.github.io#246] [Q&A] [answered]
Hi

What does this setting do? I can see a message in the kodi.log ```Trakt Sync Service Starting (sync check every 15 minutes)...``` but what does it actually do? I'm assuming that changes made to Trakt will be synced back to Umbrella? What type of changes? How can I check this works?

Thanks

**Comment by umbrellaplug** [comment, marked as answer]
This setting sets how often to sync with Trakt. It's used for scrobble times and watch history. Many use kodi from multiple devices and syncing with Trakt like this keeps the information consistent across devices.
